.github/workflows/build.yml

```diff
@@ -16,6 +16,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
+          toolchain: nightly
```

Comment:
are we bringing back the nightly?

Reply:
In this PR I change the toolchain file to be `rust-toolchain.toml`. This behaves differently and the action doesn't pick it up - see the action performed on the [penultimate commit](https://github.com/dusk-network/hatchery/actions/runs/3273676619). When running `make test` locally the correct toolchain will always be loaded.
We have to be careful in the CI just as before, but at least we will know how to solve the problem quickly.